parser_library/test/common_testing.h

```diff
@@ -148,14 +156,17 @@ std::optional<std::unordered_map<size_t, T>> get_var_vector_map(hlasm_context& c
 
 template<typename Msg,
     typename Proj,
+    typename BinPred = std::function<bool(
+        std::decay_t<std::invoke_result_t<Proj, const Msg&>>, std::decay_t<std::invoke_result_t<Proj, const Msg&>>)>,
```

Comment:
```suggestion
    typename BinPred = std::equal_to<>,
```
and then `BinPred b = BinPred()`
or
```suggestion
    std::predicate<std::decay_t<std::invoke_result_t<Proj, const Msg&>>,std::decay_t<std::invoke_result_t<Proj, const Msg&>>> BinPred = std::equal_to<>,
```